GEOSldas_App/util/postproc/ObsFcstAna_stats/helper/compute_monthly_stats.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+from datetime import datetime, timedelta
+from dateutil.relativedelta import relativedelta
+from helper.read_GEOSldas import read_ObsFcstAna, read_tilecoord, read_obs_param
+
+def compute_monthly_stats(expdir,expid,domain,this_month,tc,obs_param,var_list):
```

Comment:
Assuming this is "Step 1)" in the processing (per the introductory comment of the PR), it would probably be better to move this script up one directory level.  Or maybe provide a (very short) driver script that executes this step for a user-specified period?   Trying to separate the "generic" Steps 1) and 2) from the final Step 3) that is very dependent on the experiment and use case.

Reply:
I moved compute_monthly_stats() out of helper/. It can be used as a stand-alone script. The "driver" is toward the end after the line 
if __name__ == '__main__':